packages/connect-iframe/src/index.ts

```diff
@@ -186,7 +186,7 @@ const filterDeviceEvent = (message: DeviceEvent) => {
         'device' in message.payload ? message.payload.device.features : message.payload.features;
     if (features) {
         const savedPermissions =
-            storage.load(storage.PERMISSIONS_KEY) || storage.load(storage.PERMISSIONS_KEY, true);
+            storage.load()[storage.PERMISSIONS_KEY] || storage.load(true)[storage.PERMISSIONS_KEY];
```

Comment:
I don't like `storage.load() || storage.load(true)` but since it was done like this already we can leave it for some future refactoring. this PR is about analytics after all

Reply:
I can split it into more lines with comments.